spec/core_ext/digest/uuid_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe Digest::UUID do
+  context "clean" do
+    let(:expected) { '01234567-89ab-cdef-abcd-ef0123456789' }
+
+    it "handles a valid lowercase guid as expected" do
```

Comment:
Can you remove the `"as expected"`.  Seems redundant to me.

Reply:
Updated.